storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/pkr-operation.cpp

```diff
@@ -68,9 +70,14 @@ PKROperation::PKROperation(Uint32 noOps, RS_Buffer *reqBuffs, RS_Buffer *respBuf
     this->subOpTuples.push_back(pkOpTuple);
   }
 
-  this->ndbObject = ndbObject;
-  this->noOps     = noOps;
-  this->isBatch   = true;
+  this->ndbObject  = ndbObject;
+  this->noOps      = noOps;
+  this->isBatch    = true;
```

Comment:
To implement support for multi tx one has to ensure that BLOBs use single transaction.